salt/_modules/metalk8s_kubernetes.py

```diff
@@ -2408,17 +2408,14 @@ def node_uncordon(node_name, **kwargs):
 
 
 def get_crd_plural_from_kind(kind, **kwargs):
-    plural = None
-
     crds = customresourcedefinition(**kwargs)
 
     for crd in crds.get('items', []):
         crd_names = crd.get('spec', {}).get('names', {})
         if kind == crd_names.get('kind'):
-            plural = crd_names.get('plural')
-            break
+            return crd_names.get('plural')
 
-    return plural
+    return None
```

Comment:
Ok for being explicit, but it's not something required (by default, a Python function with no `return` statement will return `None`)

Reply:
Most linters will yell at you for mixing explicit and implicit returns in a function though.